cli/Webion.Templates.Cli/Http/TemplatesClient.cs

```diff
@@ -0,0 +1,48 @@
+using System.Net;
+using System.Text;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Webion.Templates.Cli.Model;
+using Webion.Templates.Cli.Options;
+
+namespace Webion.Templates.Cli.Http;
+
+public sealed class TemplatesClient : HttpClient
+{
+    public TemplatesClient(IOptions<TemplatesOptions> options)
+    {
+        BaseAddress = new Uri(options.Value.Url);
+    }
+
```

Comment:
In `Program.cs` dentro `builder.ConfigureServices()` c'è `services.AddHttpClient<Interface, Impl>()` (`using Microsoft.Extensions.DependencyInjection`)

Rendilo iniettabile come `HttpClient`

Deve accettare l'HttpClient da costruttore ed averlo come campo `private readonly`, ed estendere l'interfaccia con i metodi del templates client

Dopo puoi iniettarlo usando l'interfaccia